src/commands/quiz.ts

```diff
@@ -0,0 +1,91 @@
+import { Command } from '../types';
+import fetch from 'node-fetch';
+
+const MAX_QUESTIONS = 10;
+const USAGE_MESSAGE = `Format: !quiz język poziom(opcjonalny) ilość(opcjonalny).
+Dostępne wartości:
+* język: html, css, js, angular, react, git, other
+* poziom: junior, mid, senior
+* ilość: [1 - ${MAX_QUESTIONS}] - ile pytań wylosować
+`;
+
+const Levels = ['junior', 'mid', 'senior'];
+const Languages = ['html', 'css', 'js', 'angular', 'react', 'git', 'other'];
+
+const quiz: Command = {
+  name: 'quiz',
+  description: 'Odpowiedz na pytanie',
+  args: true,
+  async execute(msg, args) {
+    const [language, level, amount = '1'] = args;
+
+    const errorMsg = validateParams(language, level, amount);
+    if (errorMsg) {
+      return msg.channel.send(`${errorMsg} \`\`\`${USAGE_MESSAGE}\`\`\``);
+    }
+
+    const url = prepareUrl(language, level);
+    const result = await fetch(url);
+    const data = (await result.json()) as DevFAQResponse;
+
+    const {
```

Comment:
kompletnie zbędna linijka, wyżej sobie wyciągnij te rzeczy :)
```suggestion
    const {data: questions, meta: {total} } = await result.json() as DevFAQResponse;
```